elasticsearch-transport/lib/elasticsearch/transport/transport/http/manticore/pool.rb

```diff
@@ -0,0 +1,201 @@
+module Elasticsearch
+  module Transport
+    module Transport
+      module HTTP
+        class Manticore
+          class Pool
+            attr_reader :logger
+
+            def initialize(logger, adapter, healthcheck_path="/", urls=[], resurrect_interval=5)
+              @logger = logger
+              @state_mutex = Mutex.new
+              @url_info = {}
+              @adapter = adapter
+              @healthcheck_path = healthcheck_path
+              @stopping = false
+              @resurrect_interval = resurrect_interval
+              @resurrectionist = start_resurrectionist
+              update_urls(urls)
+            end
+
+            def close
+              logger.info  "Stopping resurrectionist" if logger
```

Comment:
I'm not sure if this, and other similar logging calls are appropriate for the `info` level? I would probably move them to the debug level?